flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutorGateway.java

```diff
@@ -32,4 +33,11 @@
 	// ------------------------------------------------------------------------
 
 	void notifyOfNewResourceManagerLeader(String address, UUID resourceManagerLeaderId);
+
+	/**
+	 * Send by the ResourceManager to the TaskExecutor
+	 * @param allocationID id for the request
+	 * @param resourceManagerLeaderID current leader id of the ResourceManager
+	 */
+	void requestSlot(AllocationID allocationID, UUID resourceManagerLeaderID);
```

Comment:
As of now, this is just a stub but we will have to acknowledge the message. Will change the signature to make that clear.